ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/blocks/BeaconBlock.java

```diff
@@ -45,72 +41,51 @@
   // Body
   private BeaconBlockBody body;
 
-  // Signature
-  private BLSSignature signature;
-
   public BeaconBlock(
-      UnsignedLong slot,
-      Bytes32 parent_root,
-      Bytes32 state_root,
-      BeaconBlockBody body,
-      BLSSignature signature) {
+      UnsignedLong slot, Bytes32 parent_root, Bytes32 state_root, BeaconBlockBody body) {
     this.slot = slot;
     this.parent_root = parent_root;
     this.state_root = state_root;
     this.body = body;
-    this.signature = signature;
   }
 
   public BeaconBlock() {
     this.slot = UnsignedLong.ZERO;
     this.parent_root = Bytes32.ZERO;
     this.state_root = Bytes32.ZERO;
     this.body = new BeaconBlockBody();
-    this.signature = BLSSignature.empty();
   }
 
   public BeaconBlock(Bytes32 state_root) {
     this.slot = UnsignedLong.ZERO;
     this.parent_root = Bytes32.ZERO;
     this.state_root = state_root;
     this.body = new BeaconBlockBody();
-    this.signature = BLSSignature.empty();
   }
 
   @Override
   public int getSSZFieldCount() {
-    return SSZ_FIELD_COUNT + body.getSSZFieldCount() + signature.getSSZFieldCount();
+    return SSZ_FIELD_COUNT + body.getSSZFieldCount();
   }
 
   @Override
   public List<Bytes> get_fixed_parts() {
-    List<Bytes> fixedPartsList = new ArrayList<>();
-    fixedPartsList.addAll(
-        List.of(
-            SSZ.encodeUInt64(slot.longValue()),
-            SSZ.encode(writer -> writer.writeFixedBytes(parent_root)),
-            SSZ.encode(writer -> writer.writeFixedBytes(state_root)),
-            Bytes.EMPTY));
-    fixedPartsList.addAll(signature.get_fixed_parts());
-    return fixedPartsList;
+    return List.of(
+        SSZ.encodeUInt64(slot.longValue()),
+        SSZ.encode(writer -> writer.writeFixedBytes(parent_root)),
+        SSZ.encode(writer -> writer.writeFixedBytes(state_root)),
+        Bytes.EMPTY);
   }
 
   @Override
   public List<Bytes> get_variable_parts() {
-    List<Bytes> variablePartsList = new ArrayList<>();
-    variablePartsList.addAll(
-        List.of(
-            Bytes.EMPTY,
-            Bytes.EMPTY,
-            Bytes.EMPTY,
-            SimpleOffsetSerializer.serialize(body),
-            Bytes.EMPTY));
-    return variablePartsList;
+    return List.of(
```

Comment:
I think the last Bytes.EMPTY here should be removed since I reckon that was for the signature

Reply:
Wow, great spot.